### ISSUE TITLE
Function `TelegramBot::updates()` don't support any parameters from Telegram API `getUpdates`

### ISSUE THREAD

**Issue by RuslanMelnychenko** [defstudio/telegraph#395]
Hello. I need local debug a telegram bot. But in this package is not designed for this possibility.
For example, I try to debug telegram bot by [`updates()`](https://github.com/defstudio/telegraph/blob/main/src/Models/TelegraphBot.php#L176), so write a command:

```php
namespace App\Console\Commands;

use DefStudio\Telegraph\DTO\TelegramUpdate;
use DefStudio\Telegraph\Models\TelegraphBot;
use Illuminate\Console\Command;
use Illuminate\Http\Request;

class TelegraphPollingUpdates extends Command
{
    /**
     * The name and signature of the console command.
     *
     * @var string
     */
    protected $signature = 'telegraph:polling-updates {bot_id?} {--interval=3 : interval for check updates}';

    /**
     * The console command description.
     *
     * @var string
     */
    protected $description = 'Polling updates';

    /**
     * Execute the console command.
     */
    public function handle()
    {
        if ($this->argument('bot_id')) {
            $bot = TelegraphBot::fromId($this->argument('bot_id'));
        } else {
            $bot_id = $this->choice(
                'Select telegram bot:',
                TelegraphBot::query()->get(['id', 'name'])->map(fn (TelegraphBot $bot) => "{$bot->id}. {$bot->name}")->toArray(),
                0
            );
            $bot = TelegraphBot::fromId(explode('.', $bot_id, 2)[0]);
        }

        $interval = (int) $this->option('interval');

        $handler = app(config('telegraph.webhook_handler'));
        while (true) {
            $result = $bot->updates();
            $result->each(function(TelegramUpdate $update) use ($handler, $bot) {
                $this->info(json_encode($update->toArray()));
                try {
                    $handler->handle(new Request($update->toArray()), $bot);
                } catch (\Exception $exception) {
                    $this->error($exception);
                }
            });
            if ($result->isEmpty()) {
                sleep($interval);
            }
        }
    }
}

```

So, it works, but function [`updates()`](https://github.com/defstudio/telegraph/blob/main/src/Models/TelegraphBot.php#L176) don't support any parameters from [Telegram API `getUpdates()`](https://core.telegram.org/bots/api#getupdates) and I can't set `offset` parameter to get next updates from telegram.

Can fix it?
Add support parameters from [`getUpdates`](https://core.telegram.org/bots/api#getupdates).

And also you can add support command `telegraph:polling-updates` from my example.

**Comment by fabio-ivona** [comment]
Hi, we'll schedule this in our next developments

meanwhile if you have pleasure, feel free to contribute by opening a PR

**Comment by fabio-ivona** [comment]
fixed in #503
